pallets/stableswap/src/lib.rs

```diff
@@ -1048,6 +1243,7 @@ impl<T: Config> Pallet<T> {
 		asset_in: T::AssetId,
 		asset_out: T::AssetId,
 		amount_in: Balance,
+		update_peg: bool, // This is needed to distinguish between just calculation and trade execution
```

Comment:
you have to recalculate peg in both cases, bonly difference is if you store it or no, therefore something like write_peg or just_calculate would be better name for this, and you woudlnt have to comment it

Reply:
renamed to persit_peg and deleted the comment